MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -825,6 +827,7 @@ private StreamInfo BuildVideoItem(MediaSourceInfo item, MediaOptions options)
             var audioBitrate = audioStream?.BitRate;
             var audioSampleRate = audioStream?.SampleRate;
             var audioBitDepth = audioStream?.BitDepth;
+            var useSubContainer = playlistItem.SubProtocol == MediaStreamProtocol.hls;
```

Comment:
I doubt that `playlistItem` is populated.
We only decide how to play.
We can probably always use a subcontainer if one exists.

Reply:
It is explicitly marked as non-nullable...

Reply:
It will be set after we decide - that's the problem.

Reply:
So ignore it here? and reert to what was there before while not considering the subprotocol?

Reply:
I would say it needs to be reverted, for now.
But it might be a good idea to just change `ContainsAnyCodec` so that it always prefers `SubContainer` if set.
We can do it in another PR.

Reply:
Hmm, we already have `playlistItem.SubProtocol == MediaStreamProtocol.hls`.

Reply:
Remove `useSubContainer` from here, for now. It is always `false`.
As gnattu said, we probably need to rethink the use of `SubContainer`.